scipy/special/tests/test_precompute_gammainc.py

```diff
@@ -36,19 +40,21 @@ def test_g():
 @check_version(sympy, '0.7')
 def test_alpha():
     # Test data for the alpha_k. See DLMF 8.12.14.
+    rtol = 2e-11 if _is_32bit_platform else 1e-17
```

Comment:
These tests are a little different from the average test since they're testing data (asymptotic series coefficients, etc.) which gets used in the special function implementations. If a platform doesn't compute the data correctly to better than machine precision then a developer should be aware of that; otherwise they will be confused as to why running the scripts in `special._precompute` doesn't recreate the data used in `special`, or even worse, they will replace the correct data with incorrect data and degrade the accuracy of the function. They are currently made aware of an issue by a test failing.

Just my opinion though.